src/styles/_catalog.scss

```diff
@@ -1,15 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  gap: $gap;
+  max-width: catalogWidth($cardWidth,
+  $gap, $cardsInRow);
   margin: 0 auto;
 
+  @media (max-width: 488px) {
+    grid-template-columns: $cardWidth;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+  }
+
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $cardWidth;
     height: 300px;
     border: 1px solid #ccc;
```

Comment:
you can add variable for this border color

Reply:
I don't want to overuse variables) 
![image](https://user-images.githubusercontent.com/123364948/217244300-a78a101f-e5f3-460d-b3f4-cf4b55efb1d7.png)